pkg/gardenlet/controller/backupbucket/backup_bucket_reconciler.go

```diff
@@ -98,26 +98,26 @@ func (r *reconciler) reconcileBackupBucket(
 	error,
 ) {
 	if !controllerutil.ContainsFinalizer(backupBucket, gardencorev1beta1.GardenerName) {
+		log.Info("Adding finalizer")
 		if err := controllerutils.StrategicMergePatchAddFinalizers(ctx, gardenClient.Client(), backupBucket, gardencorev1beta1.GardenerName); err != nil {
-			log.Error(err, "Could not add finalizer")
 			return reconcile.Result{}, err
 		}
 		return reconcile.Result{}, nil
 	}
 
 	if updateErr := updateBackupBucketStatusProcessing(ctx, gardenClient.Client(), backupBucket, "Reconciliation of Backup Bucket state in progress.", 2); updateErr != nil {
-		log.Error(updateErr, "Could not update status after reconciliation start")
 		return reconcile.Result{}, updateErr
```

Comment:
Can you move the messages from the logs you removed to the respective returned error? I.e.:
```go
return reconcile.Result{}, fmt.Errorf("Could not update status after reconciliation start: %w", updateErr)
```

The raw errors alone might not be enough to pin down the origin.